### ISSUE TITLE
endless loop in dev

### ISSUE THREAD

**Issue by jeremy-coleman** [fuse-box/react-example#21]
![image](https://user-images.githubusercontent.com/25135855/57531806-4f5df100-7308-11e9-8e18-31a921e47cf0.png)

all i did was install and start  , and updated the ts version 3.4.5

**Comment by nchanged** [comment]
hm, what are the steps so I could reproduce it? You are on v4 branch right?

**Comment by nchanged** [comment]
also, are you on windows?

**Comment by jeremy-coleman** [comment]
Yes on windows , that is the v4.10, all i did was change 1 character in the react component, it happenes immediately

**Comment by nchanged** [comment]
Yep thanks I know how to fix it. Problem with watchers ignore on windows
